ci

```diff
@@ -448,6 +448,25 @@ case "$1" in
     refresh_fstar_hints
     ;;
 
+  fstar-docs-nightly)
+    # Create the docs associated with FStar
+    if [ ! -d ulib ]; then
+      echo "I don't seem to be in the right directory, bailing"
+      exit 1
+    fi
+
+    # First - get fstar built
+    git clean -ffdx
+    git reset --hard HEAD
+    git pull origin master
```

Comment:
is F* docs a nightly run or a CI run? I'm curious why you need to pull from the latest master -- the point of using VSTS is that it already puts the git repository at the right revision, correct?